packages/sales-api-service/src/services/renewals/renewals.service.js

```diff
@@ -0,0 +1,101 @@
+import { SERVICE_LOCAL_TIME } from '@defra-fish/business-rules-lib'
+import moment from 'moment-timezone'
+
+// Replicated from GAFL - need to decide whether to move
+const cacheDateFormat = 'YYYY-MM-DD'
+const licenceToStart = {
+  AFTER_PAYMENT: 'after-payment',
+  ANOTHER_DATE: 'another-date'
+}
+
+export const preparePermissionDataForRenewal = existingPermission => {
+  return {
+    ...prepareBasePermissionData(existingPermission),
+    ...prepareDateData(existingPermission),
+    licensee: prepareLicenseeData(existingPermission),
+    permitId: preparePermitId(existingPermission)
+  }
+}
+
+const prepareBasePermissionData = existingPermission => {
+  return {
+    isRenewal: true,
+    licenceLength: '12M', // Always for renewals
+    licenceType: existingPermission.permit.permitSubtype.label,
+    numberOfRods: existingPermission.permit.numberOfRods.toString(),
+    isLicenceForYou: true
+  }
```

Comment:
this and the previous function would be slightly more succinct if you omitted the return - so do
```
export const preparePermissionDataForRenewal = existingPermission => ({
    ...prepareBasePermissionData(existingPermission),
    ...prepareDateData(existingPermission),
    licensee: prepareLicenseeData(existingPermission),
    permitId: preparePermitId(existingPermission)
})
```